node-installer/internal/constants/constants.go

```diff
@@ -14,6 +14,9 @@ import (
 )
 
 var (
+	// Version value is injected at build time.
+	Version = "0.0.0-dev"
```

Comment:
AFAICT this will use the same value for every development build. This is bad because the node-installer will skip some of its install actions if it detects that a runtime handler with the same name already exists.

Reply:
I know, this will be handled in a follow-up for pseudo-versioning

Reply:
Personally, I'd rather have this be part of the same PR and not break people working on main.

Reply:
I mean, you can still deploy, but you just need to clean up afterwards, or change the version.


I'm fine with implementing it, but want to keep the PR reviewable